docs/cookbook/adding-a-new-email-template.md

```diff
@@ -361,7 +361,51 @@ Shopsys\FrameworkBundle\Model\Mail\MailTemplateConfiguration:
               - '@=service("App\\Component\\Mail\\PasswordChangedMailTemplateVariablesProvider").create()'
 ```
 
+## Bonus: Enable testing the email template in administration on click
+
+This step is not necessary for the email template to work, but with a little effort, it could add a great value for the developers, testers, and the e-shop administrators.
+By adding a new implementation of `Shopsys\FrameworkBundle\Model\Mail\MailTemplateSender\MailTemplateSenderInterface`, it will be possible to send the email template to any desired email without the need of performing the action that normally triggers the mail sending.
```

Comment:
```suggestion
By adding a new implementation of `Shopsys\FrameworkBundle\Model\Mail\MailTemplateSender\MailTemplateSenderInterface`, it will be possible to send the email template to any desired email without the need for performing the action that normally triggers the mail sending.
```